File: LeetCode/Simulation/Python/Leetcode1791-找出星型图的中心节点.py
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        n = len(edges) + 1
        st = [0] * (n + 1)
        for e in edges:
            st[e[0]] += 1
            st[e[1]] += 1
        for i in range(1,n + 1):
            if st[i] == n - 1: return i
        return -1

File: LeetCode/Simulation/JS/Leetcode1791-找出星型图的中心节点.js
var findCenter = function(edges) {
    const n = edges.length + 1;
    const st = new Array(n + 1).fill(0);
    for(const e of edges){
        st[e[0]]++;
        st[e[1]]++;
    }
    for(let i = 1;i <= n;i++){
        if(st[i] == n - 1) return i;
    }
    return -1;
};